docs/yul.rst

```diff
@@ -945,8 +947,9 @@ the ``dup`` and ``swap`` instructions as well as ``jump`` instructions, labels a
   The remaining bytes will retain their values as of before the call.
 
 .. note::
```

Comment:
Maybe some more refinement into something like:
```
The `difficulty()` instruction is disallowed in EVM version >= Paris.  With the Paris network upgrade the semantics of the instruction that was previously called ``difficulty`` have been changed and the instruction was renamed to ``prevrandao``.  It can now return arbitrary values in the full 256-bit range, whereas the highest recorded difficulty value within Ethash was ~54 bits.

```